YomogiFont: Version 3.000; ttfautohint (v1.8.3) added

aaronbell opened the pull request:
5420cfe: [gftools-packager] YomogiFont: Version 3.000; ttfautohint (v1.8.3) added

* YomogiFont Version 3.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/satsuyako/YomogiFont.git at commit https://github.com/satsuyako/YomogiFont/commit/e7f8236fcf416b4e1fe797e109f3d4d081cc21ea.

 a73d747: [gftools-packager] ofl/yomogifont remove METADATA "source".  google/fonts#2587

m4rc1e commented:
![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/7525512/106767881-3f855500-6633-11eb-8083-9868341c44c7.png)

![Desktop_Windows_10_chrome_71 0_](https://user-images.githubusercontent.com/7525512/106767893-42804580-6633-11eb-8d56-7c61c0c7039b.png)

davelab6 commented:
Please remove "Font" from the family name. @mintoming please coordinate that with the designer.

mintoming commented:
Thanks Dave!
Will do 👍

On Thu, Feb 4, 2021 at 11:53 PM Dave Crossland <notifications@github.com>
wrote:

> Please remove "Font" from the family name. @mintoming
> <https://github.com/mintoming> please coordinate that with the designer.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/fonts/pull/2882#issuecomment-773368462>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANTCRP65FYCAIBFS4JETTD3S5KYHDANCNFSM4UUCZYZA>
> .
>


-- 
*Min-Young Kim*｜UI･UX Typography Consultant
+81 - 80 - 9404 - 0022
mintoming.com

aaronbell commented:
Updated

## Yomogi: Version 3.000; ttfautohint (v1.8.3) added

---
 d77ab61: [gftools-packager] Yomogi: Version 3.000; ttfautohint (v1.8.3) added

* Yomogi Version 3.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/satsuyako/YomogiFont.git at commit https://github.com/satsuyako/YomogiFont/commit/d78510d9e7f9df2cf9bbba0214d7fde9337c2597.

aaronbell commented:
Updated font name in the file.

RosaWagner commented on ofl/yomogi/upstream.yaml:
```diff
@@ -0,0 +1,5 @@
+branch: ver3.00
+files:
+  fonts/ttf/Yomogi-Regular.ttf: YomogiFont-Regular.ttf
```
I think it needs to be updated here too for next updates

aaronbell replied:
I've updated this in the PR and upstream.  :)

(Changing font file names midstream is fun!)

RosaWagner commented on ofl/yomogi/DESCRIPTION.en_us.html:
```diff
@@ -0,0 +1,12 @@
+The design of the letters combined cuteness and readability, and it is made into a monospaced font that could also be used in the vertical writing style.
+</p>
+<p>
+This font is licensed under the SIL Open Font License, Version 1.1.
```
Since almost all GF fonts are under OFL, we can also remove this line.

aaronbell replied:
Done.

RosaWagner commented:
the script `gftools add-font` updates metadata.pb according to what font files is in there
